readme.md

```diff
@@ -1,13 +1,69 @@
+
+### Data Hubs
+
+**An often-proposed (alleged) solution is to build data exchange hubs**: They collect individual data sets (of both plan & realtime data), integrate them – often using hand-written matching tables and fuzzy matching – and emit one large data merged set. **This doesn't work** for the following reasons:
+
+- **It doesn't scale.** Currently, transportation data hubs process data from *dozens or hundreds* of operators. In Europe, with the on-going heterogenisation of mobility, to really cover *all mobility options*, they would have to integrate *thousands to ten-thousands* of operators.
+- Almost always, **the raw & merged data is not (easily) accessible**. This makes it hard to a) integrate more data sources (because of missing openly available examples & APIs) and b) improve the merging algorithms.
+- **It is neither fair nor innovative.** Designing centralized systems (including a federation of central hubs) disproportionately helps large transportation operators and app companies (Google Maps), and actively harms small, innovative operators from appearing in people's day-to-day mobility apps.
+- **It's not resilient.** With the coming level of integration in the mobility sector, large centralized data hubs will a) affect a huge number of people when they're down/malfunctioning, b) make it harder to build offline-capable, data-saving, low-end-devices-compatible (IoT) clients, and therefore encourage the reliance on always-on cloud systems.
+
+### Linked Open Transport Data
+
+Let's solve these problems by designing our public transportation systems *from the start* **with federation, discovery of data sources and caching/offline compatibility in mind**!
+
+We must make our data
+
+- **self-descriptive** (referencing human- & machine-readable documents on how to parse & interpret it),
+- **globally precise** (using IDs that accurately identify transportation infrastructure *without local or operator-specific context*),
+- **redistributable** (being in the [public domain](https://en.wikipedia.org/wiki/Public_domain), allowing for easy chunking, replication & storage).
+
+We must make our APIs
+
+- **openly documented** (using open standards),
+- **publicly available** (without authentication, in the internet, in a best-effort approach),
+- **federated** (linking to other APIs instead of aggregating them).
+
+We must develop our **data standards** in the open (allowing barrier-free participation & collaboration) and **freely licensed** (to enable wide-spread use). They **should cleanly separate semantics, replication/transport, storage & encoding**. They should not reinvent the wheel ([NeTEx](https://en.wikipedia.org/wiki/NeTEx)), but **rely on existing de-facto standards** (such as [GTFS](https://gtfs.org)) where applicable.
```

Comment:
Transmodel is in that regards more on topic in my opinion: it is the domain model that backs NeTEx. This can be applied on Linked Data too: https://github.com/oeg-upm/transmodel-ontology